### PR TITLE
add size sort

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

**Review by ShifaSZ — CHANGES_REQUESTED** [review]

File: seqr/utils/elasticsearch/constants.py [hunk 1]
```diff
@@ -220,6 +220,14 @@
             }
         },
     }],
+    'size': [{
+        '_script': {
+            'type': 'number',
+            'script': {
+               'source': "doc['start'].value - doc['end'].value"
```
[the remark this answers is not in the document]
Comment (hanars) [reply seqr/utils/elasticsearch/constants.py:227]: yes, the negative values cause it to sort in the correct direction. And its biologically/ physically impossible for a single SV to span multiple chromosomes

[the remark this answers is not in the document]
Comment (ShifaSZ) [reply seqr/utils/elasticsearch/constants.py:227]: The translocation SVs can move from one chromosome to another one. We can find many translocation SVs have a different CHR2 from their contig.

[the remark this answers is not in the document]
Comment (hanars) [reply seqr/utils/elasticsearch/constants.py:227]: A transloacation that moves from one chromosome to another os not the "size" of the distance between chromosomes, its the size of whatever element was moved. This is the computation we use when we display translocations to users and noone has commented that its looks wrong. Moreover, we want the sort to correspond to whats displayed so it looks logical

[the remark this answers is not in the document]
Comment (ShifaSZ) [reply seqr/utils/elasticsearch/constants.py:227]: It seems that the information of a translocation SV is insufficient for its "size". We use the below code to generate the `end` field in the genome SV loading pipeline. And the END2 is just the breaking point on the second chromosome.
```
hl.if_else(hl.is_defined(rows.info.END2), rows.info.END2, rows.info.END)
```

[the remark this answers is not in the document]
Comment (hanars) [reply seqr/utils/elasticsearch/constants.py:227]: this is the size we use for display. this is the size we will be using for sort. this is not negotiable

[the remark this answers is not in the document]
Comment (ShifaSZ) [reply seqr/utils/elasticsearch/constants.py:227]: We can go ahead with this issue. It also will be better for us to give the analysts a heads up.

**Pull request merged**